Lib/xml/dom/minidom.py

```diff
@@ -300,12 +300,27 @@ def _in_document(node):
         node = node.parentNode
     return False
 
-def _write_data(writer, data):
+def _write_data(writer, text, attr):
     "Writes datachars to writer."
-    if data:
-        data = data.replace("&", "&amp;").replace("<", "&lt;"). \
-                    replace("\"", "&quot;").replace(">", "&gt;")
-        writer.write(data)
+    if text:
```

Comment:
I usually prefer representing easy special cases in a short
```suggestion
    if not text:
        return
```
rather than adding indentation to long chunks of code that only makes readers look down to see if something joined is still done at the end. Better make it clear right away that we're done handling this case.

Reply:
Agree. I just kept the style of the old code.